src/files.zig

```diff
@@ -21,6 +21,35 @@ pub fn writeZon(path: []const u8, zon: ZonElement) !void {
 	try cwd().writeZon(path, zon);
 }
 
+pub fn openDirInWindow(path: []const u8) void {
+	var newPath: []u8 = undefined;
+	defer if (builtin.os.tag == .windows) main.stackAllocator.free(newPath);
+
+	if (builtin.os.tag == .windows) {
+		newPath = main.stackAllocator.alloc(u8, std.mem.replacementSize(u8, path, "/", "\\"));
+		_ = std.mem.replace(u8, path, "/", "\\", newPath);
```

Comment:
It's only one character, you should use replaceScalar.
Also with one character `std.mem.replacementSize` is just `path.len`.